ipvs/ipvs.go

```diff
@@ -5,12 +5,16 @@ package ipvs
 import (
 	"net"
 	"syscall"
+	"time"
 
 	"fmt"
+
 	"github.com/vishvananda/netlink/nl"
 	"github.com/vishvananda/netns"
 )
 
+var netlinkSocketsTimeout = 3 * time.Second
```

Comment:
For my understanding is this a standard timeout ?

Reply:
nope is decided from us, the tradeoff is between waking up too ofter or being block too much time. Don't have an opinion on how much is the "right" time here